src/api/app/models/decision_favored_with_delete_request.rb

```diff
@@ -7,7 +7,7 @@ def description
   end
 
   def self.display_name
-    'favored with delete request'
+    'favor and delete the request'
```

Comment:
That's not what it does 

```suggestion
    'favor and create a delete request'
```

Reply:
of course :facepalm: